src/GafferML/TensorToImage.cpp

```diff
@@ -105,6 +105,40 @@ ImageShape imageShape( const Tensor *tensor, bool interleavedChannels )
 	}
 }
 
+/// \todo Would this be better off in `Tensor` as a method that can take any
+/// ONNX data type and convert to any IECore data type?
```

Comment:
I think there's a decent case for making the complete version from `Tensor.cpp` public, but that can wait. I'm curious what you mean by "convert to any IECore data type" though - would that be different than this?